src/styles/components/_form.scss

```diff
@@ -36,34 +38,63 @@
   &__input-label {
     display: block;
     margin-bottom: rem(12px);
+
+    font-size: rem(16px);
+    font-weight: 600;
+    line-height: 1.2;
   }
 
-  &__input-group {
+  &__group {
     width: 100%;
   }
 
-  &__actionable,
-  &__recaptcha {
+  &__input-password {
+    position: relative;
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    box-sizing: border-box;
   }
 
   &__input {
-    width: 100%;
+    display: block;
     box-sizing: border-box;
-    padding: rem(4px 12px);
+    width: 100%;
+    border-width: 0 0 rem(2px);
+    border-color: $havelock-blue;
+    outline: 0;
+    padding: 0 0 rem(8px);
+
+    font-size: rem(16px);
+
+    &--invalid {
+      border-color: $coral-red;
+    }
   }
 
   &__button {
-    color: $light;
-    text-decoration: none;
-    margin: 0;
     padding: rem(5px 10px);
     border-radius: rem(3px);
+    margin: 0;
+
     background-color: $shuttle-gray;
+    color: $light;
+    text-decoration: none;
 
     &:hover {
       background-color: darken($silver, 8%);
     }
   }
+
+  &__error {
```

Comment:
Since this is the helper text of the input, should we follow this by changing it to `&__help` 

<img width="817" alt="Screen Shot 2564-03-23 at 01 02 13" src="https://user-images.githubusercontent.com/32285706/112036602-6a8c1f80-8b73-11eb-9c69-836c2e7da10d.png">